docs/requirements-docs.txt

```diff
@@ -1,5 +1,5 @@
 numpydoc
-sphinx
+sphinx==3.3.1
```

Comment:
Thanks for reporting and fixing this issue upstream @ravwojdyla! Could you please add a comment that informs others when we can remove this pinning. Something like:

```suggestion
# TODO: Remove sphinx pinning once sphinx==3.4.1 is released
# See https://github.com/dask/dask/issues/7001
sphinx==3.3.1
```

Reply:
@jrbourbeau done.